man/udev.xml

```diff
@@ -514,6 +514,11 @@
             <term><varname>LABEL</varname></term>
             <listitem>
               <para>A named label to which a <varname>GOTO</varname> may jump.</para>
+
+              <para>Although <varname>LABEL</varname> keys are assignments,
+              they are processed before all other keys. In each rule only the
```

Comment:
"processed before"? what does that mean? i guess this comment makes things more confusing than enlightening, no?

Reply:
This means that LABEL, while being an assignment. takes effect regardless of other expressions, including any match conditions.
Could you suggest a better wording explaining this?